client/collectors/collectd.py

```diff
@@ -3,8 +3,8 @@
 from utils.logging import log
 from utils.misc import run_cmd
 
-COLLECTD_CONFIG = '/tmp/.collectd.conf'
-COLLECTD_PIDFILE = '/tmp/.collectd.pid'
+COLLECTD_CONFIG = '/raid/.collectd.conf'
```

Comment:
if these configs are specific to your local environment, we don't need to checkin